src/core/utils/internal/parseGraphQLRequest.ts

```diff
@@ -92,7 +93,7 @@ function extractMultipartVariables<VariablesType extends GraphQLVariables>(
 async function getGraphQLInput(request: Request): Promise<GraphQLInput | null> {
   switch (request.method) {
     case 'GET': {
-      const url = new URL(request.url)
+      const url = urlFromRequestOrCache(request)
```

Comment:
Are URL constructors really that costly? I suspect this optimization isn't worth the cost of readability and debugging.